src/Tracker/TrackerClient.php

```diff
@@ -447,7 +447,7 @@ public function trackPageview($event = '') {
         return $this->trackEvent($this->pageview_event);
     }
 
-    public function trackAction($action_group = '', $action_name, $action_label = '', $numeric_value = 0) {
+    public function trackAction($action_name, $action_group = '', $action_label = '', $numeric_value = 0) {
```

Comment:
We should fix the docs to match the code, not the other way around as this would be a breaking change for people.

Reply:
Yeh I ummed over it. Do you fix the bug and break code using the bug or leave it as is. I was going with leave it but in 8.0 this will break. So at some point you need to change it

Reply:
Oh I missed that. Are you saying that in php8 the setting of the default as the first parameter throws an error?

Reply:
In 8.0 you get a depreciated warning if you have a required parameter after an optional one.

Reply:
I see. Let's leave it the way you fixed it then and we will note the change. Both are strings so it won't throw an error but will change the labels on action events.